packages/documentation-site/patternfly-docs/pages/home.js

```diff
@@ -157,6 +159,9 @@ const FeaturedBlogCard = ({postData, idx}) => {
 
 const HomePage = () => (
   <React.Fragment>
+    <Banner color="blue" isSticky>
+      Welcome to the PatternFly 6 staging website. PatternFly 6 and the documentation are <span className="pf-v6-u-font-weight-bold">still under development.</span>
```

Comment:
```suggestion
      Welcome to the PatternFly 6 staging website! PatternFly 6 is <span className="pf-v6-u-font-weight-bold">still under development, so documentation across this website will continue to change.</span>
``` 
wdyt about something like this?